docs/how_tos/manual_testing.rst

```diff
@@ -6,30 +6,24 @@ The producer can be tested manually against a Kafka running in devstack.
 #. Make or refresh a copy of this repo where it can be seen from inside devstack: ``rsync -sax -delete ./ ../src/event-bus-kafka/``
 #. In devstack, start Kafka and the control webapp: ``make dev.up.kafka-control-center`` and watch ``make dev.logs.kafka-control-center`` until server is up and happy (may take a few minutes; watch for ``INFO Kafka startTimeMs``)
 #. Load the control center UI: http://localhost:9021/clusters and wait for the cluster to become healthy
-#. In edx-platform's ``cms/envs/common.py``:
-
-   - Add ``'edx_event_bus_kafka'`` to the ``INSTALLED_APPS`` list
-   - Add the following::
-
-       EVENT_BUS_KAFKA_CONSUMERS_ENABLED = True
-       EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS = "edx.devstack.kafka:29092"
-       EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL = "http://edx.devstack.schema-registry:8081"
+#. In edx-platform's ``cms/envs/common.py``, add ``'edx_event_bus_kafka'`` to the ``INSTALLED_APPS`` list
```

Comment:
I don't think this is necessary. I never do it.

Reply:
Oh, no -- it's necessary for running the management commands, which is the approach these instructions use.